### ISSUE TITLE
[6.x] Certificate default template: Validation error for the Spacer block in WordPress 5.8 block editor

### ISSUE THREAD

**Issue by eri-trabiccolo** [gocodebox/lifterlms#2024]
### Reproduction Steps

+ with wp 5.8
+ create a new certificate template
+ look at the console log

### Expected Behavior

+ No validation error

### Actual Behavior

+ Validation error

### Error Messages / Logs

```
blocks.min.js?ver=e2b6602c7ebbe8ce93832ce35d81be1c:2 Block validation: Block validation failed for `core/spacer` (Object).

Content generated by `save` function:

<div style="height:100px" aria-hidden="true" class="wp-block-spacer"></div>

Content retrieved from post body:

<div style="height:5px" aria-hidden="true" class="wp-block-spacer"></div>
```

### This issue has been recreated:
+ [x] Locally
+ [ ] On a staging site
+ [ ] On a production website
+ [ ] With only LifterLMS and a default theme

### Browser, Device, and Operating System Information
n/a

**Comment by thomasplevy** [comment]
It looks like the height attribute used to be a number and is now a string (with pixels included) but I'm not sure that explains this warning exactly...

I'm going to dump this off me and someone else can review in greater detail to come up with a solution.

**Comment by pondermatic** [comment]
This issue happens even when editing a standard WordPress post in 5.8.3 and 5.9.0. It is not caused by LifterLMS and is fixed in WordPress 5.9.1.

While it is true that the spacer block attributes changed from a number in WordPress [5.9.0](https://github.com/WordPress/wordpress-develop/blob/5.9.0/src/wp-includes/blocks/spacer/block.json#L9-L12) to a string in [5.9.1](https://github.com/WordPress/wordpress-develop/blob/5.9.1/src/wp-includes/blocks/spacer/block.json#L9-L12), this issue is caused by the style attribute `height:5px` from [wordpress.org](http://api.wordpress.org/patterns/1.0/) being different than `height:100px` from [spacer](https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/blocks/spacer/block.json) via a few JavaScript reformatting functions.

`@wordpress/blocks` v11.1.5, first used in WordPress 5.9.0, will [migrate](https://github.com/WordPress/gutenberg/blob/%40wordpress/blocks%4011.1.5/packages/blocks/src/api/parser/apply-block-deprecated-versions.js#L61-L68) deprecated block attributes, which seems to take care of difference in attribute values. `@wordpress/block-library` v6.0.26, first used in WordPress 5.9.1, adds the [deprecated definition](https://github.com/WordPress/gutenberg/blob/%40wordpress/block-library%406.0.26/packages/block-library/src/spacer/deprecated.js) to the spacer block.

| WordPress | blocks | block-library |
|-------------|--------|----------------|
| 5.8.3 | 9.1.8 | 3.2.19 |
| 5.9.0 | 11.1.5 | 6.0.25 |
| 5.9.1 | 11.1.5 | 6.0.26 |

BTW, I love being able to easily switch between different version of WordPress. I'm still using the `src` releases from the repository, but getting the code from git (actually a fork on pondermatic) instead of svn allows me to commit the built assets and switch/checkout different versions of WordPress. Building the assets can sometimes be a pain on Windows. This way, I only have to do it once per WP release.

**Comment by davidgalvez** [comment]
I had the same error message in cosole, it was resolved updating wordpress from 5.8 to 6.0
